### PR TITLE
feat(site): Added categories

### PR DESCRIPTION
**Pull request by Dacit**
Added categories to shortlinks page, for better readability.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -2,13 +2,18 @@
...
+{% assign categories = site.shortlinks | group_by:"category" | sort:"name" %}
+
+{% for group in categories %}
+<h2>{{ group.name }}</h2>
 <ul>
-  {% for shortlink in site.shortlinks %}
+    {% for shortlink in group.items %}
     <li>
-      <a href="{{ shortlink.redirect }}">{{ shortlink.title | slugify }}</a>: {{ shortlink.description }}
+        <a href="{{ shortlink.redirect }}">{{ shortlink.title | slugify }}</a>: {{ shortlink.description }}
```
Comment (kappelmann) [inline index.md:12]:
```suggestion
        <a id="{{ shortlink.title | slugify }}" href="{{ shortlink.redirect }}">{{ shortlink.title | slugify }}</a>: {{ shortlink.description }}
```

File: _shortlinks/home.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
 redirect: https://isabelle.in.tum.de
 description: The official website and download page.
+category: Infrastructure
```
Comment (kappelmann) [inline _shortlinks/home.md:4]: Why is this `Infrastructure` and not `Resources`?

Comment (Dacit) [reply _shortlinks/home.md:4]: Since the documentation is linked separately as resource and the home-page mainly serves as download site.

Comment (kappelmann) [reply _shortlinks/home.md:4]: I would have said that the download website is more like a resource (it gives something to the user) than being part of the infrastructure (it provides something to developers). Or how would you frame the two categories?

**Pull request merged**